src/main.js

```diff
@@ -4,19 +4,20 @@ import VueResource from 'vue-resource';
 
 import App from './App.vue';
 import store from './store';
+import Config from './config';
 import { routes } from './routes';
 
 Vue.use(VueRouter);
 Vue.use(VueResource);
 
-Vue.http.options.root = 'https://stocktrading-vuejs.firebaseio.com/';
+Vue.http.options.root = Config.firebase.rootUrl;
```

Comment:
Nice change! Hopefully, I will change all the literals for the config params soon. 💪